database/meta.go

```diff
@@ -355,8 +355,8 @@ func (m *MetaDB) MarkDBAnalyzed(name string, complete bool) error {
 	_, err = ssn.DB(m.config.S.MongoDB.MetaDB).C(m.config.T.Meta.DatabasesTable).
 		Upsert(bson.M{"_id": dbr.ID}, bson.M{
 			"$set": bson.D{
-				{"analyzed", complete},
-				{"analyze_version", versionTag},
+				{Name: "analyzed", Value: complete},
+				{Name: "analyze_version", Value: versionTag},
```

Comment:
Linter fixes. The linter doesn't like it when you use define a struct without listing the key names. Fixed it up since the linter wouldn't quit yelling after I changed the lines below.